lib/Yam/agama/agama_base.pm

```diff
@@ -29,8 +29,7 @@ sub post_run_hook {
     reset_consoles;
     $testapi::username = "bernhard";
     $testapi::password = 'nots3cr3t';
-    select_serial_terminal();
-    ensure_serialdev_permissions;
+    enter_cmd 'reboot';
```

Comment:
the reboot should be in the test, not in the base class, the base class only setup things, reboot is a functional part of the test, it could even done through the UI.